### ISSUE TITLE
acorn-import-meta produces a warning on installing javascript-obfuscator

### ISSUE THREAD

**Issue by cekvenich** [javascript-obfuscator/javascript-obfuscator#554]
When installing any module that uses the latest javascript-obfuscator I get a warning:

`npm WARN acorn-import-meta@1.0.0 requires a peer of acorn@^6.0.0 but none is installed. You must install peer dependencies yourself.`

I think that package would like you to use older version of acorn.

**Comment by sanex3339** [comment]
Issue
Issue

**Comment by sanex3339** [comment]
The fix was published to `acorn-import-meta`, so I'll fix it on the obfuscator side soon

https://github.com/acornjs/acorn-import-meta/issues/3

**Comment by sanex3339** [comment]
A fix in master. Expect a release later today.

**Comment by sanex3339** [comment]
Released as `0.28.0`

**Comment by cekvenich** [comment]
I get the same warning with the new .28 release. 
npm WARN acorn-import-meta@1.1.0 requires a peer of acorn@^6.0.0 || ^7.0.0 but none is installed. You must install peer dependencies yourself.

└─┬ javascript-obfuscator@0.28.0
  └── acorn-import-meta@1.1.0 

The way I 'fix it' is to include        "acorn": "^6.4.1" in dependencies.

**Comment by sanex3339** [comment]
Probably something wrong with your package-lock.json?
Tried to install with a clear project:

```
MacBook-Air-S:acorn-import-meta-test sanex$ npm i javascript-obfuscator --save

> javascript-obfuscator@0.28.0 postinstall /Users/***/***/acorn-import-meta-test/node_modules/javascript-obfuscator
> opencollective || exit 0

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN acorn-import-meta-test@1.0.0 No description
npm WARN acorn-import-meta-test@1.0.0 No repository field.

+ javascript-obfuscator@0.28.0
added 67 packages from 94 contributors and audited 73 packages in 9.433s

2 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by cekvenich** [comment]
@sanex3339 nope. I deleted package-lock.json and did npm rebuild.
npm i 
still reports same warning

**Comment by sanex3339** [comment]
Released `0.28.1` and completely remove `acorn-import-meta` package.

**Comment by cekvenich** [comment]
I confirm that the fix works for me.
